### PR TITLE
Bump operator to v1.1.13 and tidb components to v4.0.13

### PR DESCRIPTION
**Pull request by KanShiori**
### What problem does this PR solve?

Bump operator to v1.1.13 and tidb components to v4.0.13

### What is changed and how does it work?

ref #3920

run script `hack/bump-version.sh`, and correct some comments by hand.

### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [x] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [x] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
None
```

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
LGTM

File: hack/bump-version.sh [hunk 1]
```diff
@@ -22,10 +22,10 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
 fi
 
 # parameters
-OPERATOR_OLD="v1\.1\.11"
-OPERATOR_NEW="v1\.1\.12"
-TIDB_OLD="v4\.0\.10"
-TIDB_NEW="v4\.0\.12"
+OPERATOR_OLD="v1\.1\.12"
+OPERATOR_NEW="v1\.1\.13"
+TIDB_OLD="v4\.0\.12"
```
Comment (DanielZhangQD) [inline hack/bump-version.sh:27]: Please also cover the DM version in `examples` and `manifests`, and update to v2.0.4.

Comment (KanShiori) [reply hack/bump-version.sh:27]: cf07ec4edd372c1d98b8d6aba11f9b0663743e13

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**